From 25aaae987bc8a73b8cce0952f87e79c20c05350c Mon Sep 17 00:00:00 2001
From: Heinrich Toews <heinrich.toews@wago.com>
Date: Wed, 26 Sep 2018 13:36:12 +0000
Subject: [PATCH] defconfig: pfc200: mlo: remove MMU_EARLY

Barebox is not able to start a former version which has CONFIG_MMU disabled
when CONFIG_MMU_EARLY is enabled. As we dont need this option we just
turn it off.

Barebox 2013 hungs at the function omap3_gp_romcode_call() in
arch/arm/mach-omap/romcode_calls.S:

The following ASM code causes a problem:

        880382e4:       e1600070        smc     0

The code tries to do a Secure Monitor Call which fails and ends
up in a vector loop.

Signed-off-by: Heinrich Toews <heinrich.toews@wago.com>
---
 arch/arm/configs/am35xx_pfc_mlo_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/am35xx_pfc_mlo_defconfig b/arch/arm/configs/am35xx_pfc_mlo_defconfig
index beadb21..bc9fdaa 100644
--- a/arch/arm/configs/am35xx_pfc_mlo_defconfig
+++ b/arch/arm/configs/am35xx_pfc_mlo_defconfig
@@ -5,6 +5,7 @@ CONFIG_THUMB2_BAREBOX=y
 # CONFIG_ARM_EXCEPTIONS is not set
 # CONFIG_MEMINFO is not set
 CONFIG_MMU=y
+# CONFIG_MMU_EARLY is not set
 CONFIG_STACK_SIZE=0xc00
 CONFIG_MALLOC_SIZE=0x0
 CONFIG_MALLOC_DUMMY=y
-- 
2.7.4

